### PR TITLE
Updated New Stories for launch day

### PR DESCRIPTION
**Pull request by Hampshire100**

### CODE REVIEW

File: fragments/home/news.yml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   href: https://www.accesstocare.va.gov/
   description: Get wait times and compare quality and patient-satisfaction data for VA health facilities.
-- img: /img/homepage/home-3.jpg
-  alt: Close-up of an American flag patch on the sleeve of a U.S. military combat uniform.
-  title: VA Rapid Appeals
-  href: https://www.blogs.va.gov/VAntage/48141/vas-rapid-appeals-modernization-program-ramp-now-open-appeals/
-  description: Find out how to get a faster decision on a pending disability compensation appeal through the VA Rapid Appeals Modernization Program (RAMP).
+- img: /img/homepage/hp-vantage.jpg
+  alt: Graphic for the VAntage Point
+  title: VAntage Point: News You Can Use
```
Comment (bshyong) [inline fragments/home/news.yml]: Typo here

**Comment by ncksllvn** [comment]
Hey @Hampshire100! I had to make a change to you pull request. The content was good, but if it contains a colon, we have add single quote before and after it. This is because a colon has a special meaning for files with the `.yml` extension. Here's an example -

```
href: 'https://youtu.be/DEfRlL63nCg'
```

**Comment by ncksllvn** [comment]
@Hampshire100 can you click the "View Deployment" button on this page and confirm that the site looks okay? 

image

**Comment by Hampshire100** [comment]
Hey Nick!
Just checked it out and it looks great. Thanks for letting me know about the syntax issue I created. 
Thank you!!!

**Pull request merged**